lib/open_api_spex/cast.ex

```diff
@@ -119,8 +119,12 @@ defmodule OpenApiSpex.Cast do
   @spec cast(t()) :: {:ok, term()} | {:error, [Error.t()]}
 
   # Custom validator
-  def cast(%__MODULE__{schema: %{"x-validate": module}} = ctx) when module != nil,
-    do: module.cast(ctx)
+  def cast(%__MODULE__{schema: %{"x-validate": module}} = ctx)
+      when is_atom(module) and module != nil,
+      do: module.cast(ctx)
+
+  def cast(%__MODULE__{schema: %{"x-validate": module}} = ctx) when is_binary(module),
+    do: module |> Elixir.String.split(".") |> Module.concat() |> apply(:cast, [ctx])
```

Comment:
Let's fix this warning, otherwise LGTM.

Reply:
@zorbash pinging you case you don't get the workflow approval notification :)